multi-bam-quantify.wdl

```diff
@@ -27,11 +27,14 @@ import "tasks/stringtie.wdl" as stringtie_task
 
 workflow MultiBamExpressionQuantification {
     input {
-        Array[Pair[String,IndexedBamFile]]+ bams #(sample, (bam, index))
+        Array[Pair[String, IndexedBamFile]]+ bams # (sample, (bam, index)).
         String outputDir = "."
         String strandedness
-        File? referenceGtfFile # Not providing the reference gtf will have stringtie do an unguided assembly
         Boolean detectNovelTranscripts = if defined(referenceGtfFile) then false else true
+        Boolean runStringtie = true
```

Comment:
There are two steps for which stringtie is used, one is controlled by `detectNovelTranscripts` and this one will control the other step (the quantification).
```suggestion
        Boolean runStringtieQuantification = true
```

Reply:
Ah yes, I did not pay attention to that. Good shout!